orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/ClientSession.java

```diff
@@ -47,7 +45,7 @@ public ClientSession(
         this.creationDate = creationDate;
         this.vtrList = vtrList;
         if (vtrList.size() > 0) {
```

Comment:
Sonar thinks this should be
```suggestion
        if (!vtrList.isEmpty()) {
```
I don't really mind, but don't want to argue with Sonar today